db/models/scores_justifications.py

```diff
@@ -0,0 +1,80 @@
+import uuid
+from datetime import datetime
+
+from db import db
+
+
+class ScoresJustifications(db.Model):
+    scores_justifications_id = db.Column(
+        "scores_justifications_id",
+        db.Text(),
+        default=lambda: uuid.uuid4(),
+        primary_key=True,
+    )
+    timestamp = db.Column("timestamp", db.Text(), default=str(datetime.now()))
```

Comment:
Can we use https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.DateTime for this? And default to `datetime.utcnow` and call it `created_at`?